### PR TITLE
Some minor fixes and a new sample application

### PR DESCRIPTION
**Pull request by NasaGeek**

### CODE REVIEW

File: map/test/test_fids.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import sys
```
Comment (maryryang2) [inline map/test/test_fids.py:1]: Was this test file accidentally coped over from fids/test/test_fids.py? If so let's delete this file and think about adding tests for this in the future

**Review by maryryang2 — COMMENTED** [review]
Hey Chris, this looks great! Might be good to add some info about this app to the readme?

Also, what ordering guarantees are you trying to resolve with removing the position topic?

File: map/frontend/index.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+		const active_flights = new Map();
+		const completed_flights = new Set();
+		const info_windows = new Set();
+
+		function initMap() {
+			map = new google.maps.Map(document.getElementById("map"), {
+				center: { lat: 0, lng: 0 },
+				zoom: 4,
+			});
+			const start_live = {{ startlive|tojson }};
```
[the remark this answers is not in the document]
Comment (NasaGeek) [reply map/frontend/index.html:36]: It's basically a special template function that will convert some python structure to JSON. In this case I'm just using it to get us from a Python boolean to a JS boolean.

**Comment by NasaGeek** [comment]
My ordering guarantee change ensures that all events for a given flight ID will arrive in the order they were produced by Firehose. Kafka doesn't provide such a guarantee for messages that aren't on the same "partition" (in kafka's terms). We use the flight ID as the key when publishing messages to the kafka topics to ensure that messages with the same flight ID share a partition. Multiple topics will always be on separate partitions, though, so to get position messages and other messages onto the same partitions, they must also be on the same topic.

**Review by maryryang2 — APPROVED** [review]

**Pull request merged**